html.json

```diff
@@ -0,0 +1,55 @@
+{
+  "deliverables": [
+    {
+      "name": "HTML5",
+      "context": {
+        "inputs": ["userguide.ditamap"],
+        "profiles": {
+          "ditavals": ["resources/html.ditaval"]
+        }
+      },
+      "output": ".",
```

Comment:
This means "output to the directory that's passed in with `-o` on the CLI". This could also be `./` or it could be omitted.